### ISSUE TITLE
Combining publicUrl with asset path gives incorrent URL

### ISSUE THREAD

**Issue by akim-bow** [laynezh/vite-plugin-lib-assets#20]
This setup
```
libAssetsPlugin({
    include: ['**/*.wasm*'],
    publicUrl: 'https://unpkg.com/'
})
```
And import
`import wasmDep from './assets/dep.wasm'`
Gives you
`const wasmDep = 'https:/unpkg.com/assets/dep.wasm'`

Which is incorrect URL and will work unexpectedly at runtime.

**Comment by akim-bow** [comment]
Tested on 0.5.1.
Probably it's because of this line
`return `export default '${path.posix.join(publicUrl, assetPath)}'``
Not a good idea to use path.join for url's

**Comment by coder-layne** [comment]
I just released v0.5.2. You can try it out.
